.github/workflows/codeql.yml

```diff
@@ -0,0 +1,70 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
Please delete this file, we already run CodeQL, see https://github.com/github/codeql/blob/ef227a47219aef17c62bfe881b6459ef20ceb828/.github/workflows/codeql-analysis.yml .